mozetl/shield/privacy_prefs.py

```diff
@@ -31,11 +32,11 @@ def transform_shield_pings(sqlContext, pings):
 
 
 def include_shield_pings(ping):
-    return ping['payload/payload/study'] == SHIELD_ADDON_ID
+    return ping['payload/study'] == SHIELD_ADDON_ID
 
 
 def etl_job(sc, sqlContext, **kwargs):
-    return testpilot_etl_boilerplate(
+    return shield_etl_boilerplate(
         transform_shield_pings,
         's3n://telemetry-parquet/harter/privacy_prefs_shield/v1'
-    )
+    )(sc, sqlContext)
```

Comment:
Should we pass the target `submission_date` through from Airflow to allow backfills?